fdbserver/ClusterController.actor.cpp

```diff
@@ -802,6 +802,20 @@ class ClusterControllerData {
 			return false;
 		}
 
+		if(!clusterControllerProcessId.present()) {
+			return false;
+		}
+
+		auto ccWorker = id_worker.find(clusterControllerProcessId.get());
```

Comment:
It looks like id_worker has `Optional<Standalone<StringRef>>` as its key, so I don't think we need to call `.get()`.